### ISSUE TITLE
js-source with multiline strings contain '//' will be wrongly striped

### ISSUE THREAD

**Issue by yuchangyuan** [ghcjs/ghcjs#778]
When a javascript file has a multline string contains '//', remain string after '//' will be wrongly striped.

For below script in 'js-sources':

```
var zz = `
abc//dd
`
```

after process by ghcjs, the final all.js will only left:
```
var zz = `
abc
`
```

**Comment by luite** [comment]
Yup this was a common problem because the C preprocessor was used on all JS files before including them in the result. Unfortunately the C preprocessor doesn't know JavaScript syntax, which would cause the files to break.

Starting with version 8.8, the compiler doesn't run the preprocessor on `.js` files anymore for this reason.

For those who do really need the C preprocessor: rename the file to `.js.pp` (that works fine in `js-sources` or on the command line)
